### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"

### DISCUSSION THREAD

**Discussion by Chrishalaby** [community/community#12083] [Copilot Conversations] [answered]
After upgrading from windows 8 to 10, I only kept the files but deleted the apps, upon redownloading Vs Code, Github copilot was giving me this error, tried reinstalling it but didn't help.

**Comment by iamzaidsoomro** [comment]
Try synching your GitHub account with your Microsoft account, and then sign into VS Code using your GitHub account, and Microsoft Account as well, this may solve this issue.

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by AbhimanyuAryan** [reply]
this worked for me

**Reply by AbhimanyuAryan** [reply]
basically the last time something went wrong during token `getting` and I wasn't sure how to allow copilot to reask for token :)
